go/teams/teams.go

```diff
@@ -384,6 +384,34 @@ func (t *Team) AllApplicationKeys(ctx context.Context, application keybase1.Team
 	return AllApplicationKeys(t.MetaContext(ctx), t.Data, application, t.chain().GetLatestGeneration())
 }
 
+func (t *Team) AllApplicationKeysWithKBFS(ctx context.Context, application keybase1.TeamApplication) (res []keybase1.TeamApplicationKey, err error) {
+	teamKeys, err := t.AllApplicationKeys(ctx, application)
+	if err != nil {
+		return res, err
+	}
+	kbfsKeys := t.KBFSCryptKeys(ctx, application)
```

Comment:
@maxtaco Are you ok with this update to this PR? This implements what we talked about last night.